cumulusci/tasks/salesforce/install_package_version.py

```diff
@@ -104,6 +104,12 @@ def _init_options(self, kwargs):
                 self.project_config,
                 get_resolver_stack(self.project_config, "production"),
             )
+        elif isinstance(version, (float, int)):
```

Comment:
I don't suppose it makes much difference whether we do it here or in `_init_options`, does it?

Reply:
You mean in `_run_task()`? I don't think so. And it always freezes to a string already, so this only affects MetaCI and local runs.

Reply:
No, I was wrong. This _is_ `_init_options()`, just a bit... longer than average.